src/components/@core/navigation-menu/light/right-menu/items.tsx

```diff
@@ -100,6 +100,11 @@ export default [
         name: "activity",
         to: "/activityFeed/list"
       },
+      {
+        active: SITE_CONFIG.DOWNLOAD_LOGS.ACTIVE,
+        name: "download_logs",
+        to: "/user/list/download-logs"
```

Comment:
also URL can be `/user/download-logs/list` since `/user/list` is bit misleading pattern if you look at `observation/list` etc.

if changing then please change this everywhere incl. component directories